test/modules/ElasticRewards/Rewards.test.mjs

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import ethers from 'ethers';
+import build from '../../../dist/index.js';
+
+const { isAddress, isHexString } = ethers.utils;
+const { assert } = chai;
+
+describe('Reward', () => {
+  const raw =
+  `{
+    "blockNumber":13441870,
+    "for":{
+      "action":"Transfer",
+      "item":{
+          "type":"Transfer",
+          "uuid":"a23b1f18-9199-44c7-99a7-4b72f580601c"
+      },
+      "message":"Transfer 0.1 from 0xE16584424F34380DBf798f547Af684597788FbC7 to 0xe16584424f34380dbf798f547af684597788fbc7"
+    },
+    "from":"0xe16584424f34380dbf798f547af684597788fbc7",
+    "hash":"QmRowqfrJZNnEtYsw4vWPMd97kLsqJPfb8scUWKnu2BCZ2",
+    "k":"100",
+    "lambda":"0.000271078620754349",
+    "m":"3.688966681390180631",
+    "to":"0xe16584424f34380dbf798f547af684597788fbc7",
```

Comment:
we should modify this string to be a reward that isn't a transfer to the same address to aid in testing some of this.